.github/workflows/build.yml

```diff
@@ -277,36 +278,37 @@ jobs:
           context: .
           file: ./Dockerfile
           outputs: type=docker,dest=dist/image.tar
-          tags: ${{ env.IMAGE_NAME }}:latest  # not to be pushed
+          # not to be pushed
+          tags: ${{ env.IMAGE_NAME }}:latest
           # For a list of pre-defined annotation keys and value types see:
           # https://github.com/opencontainers/image-spec/blob/master/annotations.md
           labels: "\
             org.opencontainers.image.created=${{
-              needs.prepare.outputs.created }}
+            needs.prepare.outputs.created }}
 
             org.opencontainers.image.description=${{
-              fromJson(needs.prepare.outputs.repometa).description }}
+            fromJson(needs.prepare.outputs.repometa).description }}
 
             org.opencontainers.image.licenses=${{
-              fromJson(needs.prepare.outputs.repometa).license.spdx_id }}
+            fromJson(needs.prepare.outputs.repometa).license.spdx_id }}
 
             org.opencontainers.image.revision=${{ github.sha }}
 
             org.opencontainers.image.source=${{
-              fromJson(needs.prepare.outputs.repometa).clone_url }}
+            fromJson(needs.prepare.outputs.repometa).clone_url }}
 
             org.opencontainers.image.title=${{
-              fromJson(needs.prepare.outputs.repometa).name }}
+            fromJson(needs.prepare.outputs.repometa).name }}
 
             org.opencontainers.image.url=${{
-              fromJson(needs.prepare.outputs.repometa).html_url }}
+            fromJson(needs.prepare.outputs.repometa).html_url }}
 
             org.opencontainers.image.version=${{
-              needs.prepare.outputs.source_version }}"
+            needs.prepare.outputs.source_version }}"
```

Comment:
Same as above regarding unnecessary changes from the skeleton- a bit annoying in cases like this since we all use different editors that may auto-adjust spacing, but if you catch it before you commit them, it's not too bad.

Reply:
Fixed in https://github.com/cisagov/domain-manager-ui/pull/247/commits/a1d42e74984490fdf467f0f56f7d83180f40f24c 👍